centreon/www/include/configuration/configObject/contact/DB-Func.php

```diff
@@ -197,6 +197,49 @@ function disableContactInDB($contact_id = null, $contact_arr = array())
     }
 }
 
+/**
+ * Unblock contacts in the database
+ *
+ * @param int|array|null $contact Contact ID, array of contact IDs or null to unblock all contacts
```

Comment:
Precise phpdoc for arrays when possible.
```suggestion
 * @param int|array<int, string>|null $contact Contact ID, array of contact IDs or null to unblock all contacts
```